CONTRIBUTING.md

```diff
@@ -186,19 +168,34 @@ directory no matter where you are in the project directory:
 $(yarn bin)/mocha -g 'D3 Tree'
 ```
 
-For the names used by the grep option, see the `name` attribute in the
-`tests` array at the top of the `tests/codepen-project.js` file.
+For the names used by the grep option, see the `name` attribute in the default
+export object of the `src/project-tests/index.js` file.
+
+**Running tests without headless mod**
 
-**Running Chrome in a headless environment**
+By default tests run in the headless mod. If you need to see how it works on
+your screen, you need to disable the headless mod. For this run the tests like
+this:
 
-Although it looks kind of magic to see the automated tests run inside Chrome the
-first few times, and it can be useful for debugging, it gets annoying quickly
-because the browser popping up interferes with other work.
+```
+HEADLESS=0 yarn test
+```
+
+If everything is set up correctly, you should see the following happen
+automatically:
+1. Browser starts.
+1. The URL for one of the sample CodePen projects loads.
+1. Layout is changed for better viewing.
+1. External javascript settings are changed to use your local copy of bundle.js.
+1. The CodePen is re-run to get the new bundle.js.
+1. The "Run Tests" button is clicked and the script waits for test to complete.
+1. The "Tests" button is clicked to see the test output.
+1. A screenshot of the results is saved.
+1. Move on to the next CodePen by returning to step 1.
 
-We tried to get headless Chrome (very new as of July 2017) to work but it had a
-lot of problems.
+**Running tests on a virtual screen**
 
-In the meantime, you can use the tried and trusted method of running a virtual
+You can use the tried and trusted method of running a virtual
```

Comment:
I believe this is an alternative to headless mode? If so, we should let people know it's optional. Something like "If for some reason headless mode doesn't work for you, you can use the tried and trusted method of running a virtual..."